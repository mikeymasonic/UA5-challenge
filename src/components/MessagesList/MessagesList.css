.TopMessage {
  color: black;
  background-image: url('https://pbs.twimg.com/profile_images/524717947931996160/g0kMdxBX.png');
}

.BottomMessage {
  color: blue;
}
