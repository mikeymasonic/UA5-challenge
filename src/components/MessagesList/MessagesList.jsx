import React from 'react';
import PropTypes from 'prop-types';
import styles from './MessagesList.css';

const MessagesList = ({ message, attribution, messageColor }) => {
  return (
    <div className={styles.MessagesList}>
      <blockquote>
        <p className={styles.Message} style={{ color: `${messageColor}` }}>
          {message}
        </p>
        <footer>
          <p className={styles.Attribution}>
            ——
            <br />
            {attribution}
          </p>
        </footer>
      </blockquote>
    </div>
  );
};

MessagesList.propTypes = {
  message: PropTypes.string,
  attribution: PropTypes.string,
  messageColor: PropTypes.string,
};

export default MessagesList;
