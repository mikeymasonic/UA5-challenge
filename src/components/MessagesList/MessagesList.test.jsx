import React from 'react';
import { shallow } from 'enzyme';
import MessagesList from './MessagesList';

describe('MessagesList component', () => {
  it('renders MessagesList', () => {
    const wrapper = shallow(<MessagesList />);
    expect(wrapper).toMatchSnapshot();
  });
});
