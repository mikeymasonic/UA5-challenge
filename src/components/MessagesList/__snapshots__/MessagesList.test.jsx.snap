// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MessagesList component renders MessagesList 1`] = `
<div
  className="MessagesList"
>
  <blockquote>
    <footer>
      <h4
        className="Attribution"
      />
    </footer>
  </blockquote>
</div>
`;
